common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -258,32 +197,41 @@ $link-color: rgb(26, 161, 222) !default;
 
 .sequence-nav-button {
   @extend %ui-depth3;
-  @include transition(all .2s $ease-in-out-quad 0s);
-  position: absolute;
   display: block;
   top: 0;
-  width: ($baseline*2);
-  height: 46px;
-  padding: 0;
+  height: 100%;
+  background: none;  // overrides default button gradient
+  background-color: white;
+  border-color: $seq-nav-border-color;
+  min-width: 120px;
+  font-weight: normal;
+  padding: 0 $baseline;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow: hidden;
+
+  span {
+    @include padding-right($baseline / 2);
+  }
 
   &.button-previous {
-    @include border-radius(35px, 0, 0, 35px);
     @include left(0);
+    @include border-radius(3px 0 0 3px);
```

Comment:
I have run into the issue that RTL does not transform the border radius correctly. 
<img width="852" alt="screen shot 2016-10-03 at 1 56 12 pm" src="https://cloud.githubusercontent.com/assets/16127998/19047760/2c2e67ec-8971-11e6-9b1e-da4fea1cd14e.png">